ui/src/app/applications/components/application-resource-tree/application-resource-tree.test.tsx

```diff
@@ -21,4 +21,51 @@ Namespace: my-ns
 Name: my-name
 Images:
 - my-image:v1`)
-});
\ No newline at end of file
+});
+
+/*
+test("compareNodes", () => {
```

Comment:
`compareNodes` isn't an exported function, but to test the function one can add `export` to it and then add it to the imports and then uncomment this code.

Reply:
Thanks for prepare test! Please export function and uncomment the test.